specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/DatabaseAdvancedThreatProtection.json

```diff
@@ -0,0 +1,292 @@
+            "schema": {
+              "$ref": "#/definitions/DatabaseAdvancedThreatProtection"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 UpdateNotAllowedOnPausedDatabase - User attempted to perform an update on a paused database.\n\n * 400 AdvancedThreatProtectionInsufficientDiskSpace - Insufficient disk space to save the Advanced Threat Protection state metadata in the database\n\n * 400 InvalidDatabaseAdvancedThreatProtectionCreateRequest - The create database Advanced Threat Protection request does not exist or has no properties object\n\n * 400 AdvancedThreatProtectionInvalidUserSuppliedParameter - An invalid parameter value was provided by the client\n\n * 400 UpsertDatabaseAdvancedThreatProtectionFailed - An error has occurred while saving Advanced Threat Protection settings, please try again later\n\n * 400 UpsertDatabaseAdvancedThreatProtectionFailed - An error has occurred while saving Advanced Threat Protection settings, please try again later\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 DatabaseDoesNotExist - User has specified a database name that does not exist on this server instance.\n\n * 404 UpsertDatabaseAdvancedThreatProtectionFailed - An error has occurred while saving Advanced Threat Protection settings, please try again later\n\n * 409 DatabaseAdvancedThreatProtectionInProgress - Setting the database Advanced Threat Protection is already in progress\n\n * 409 UpsertDatabaseAdvancedThreatProtectionFailed - An error has occurred while saving Advanced Threat Protection settings, please try again later\n\n * 500 DatabaseIsUnavailable - Loading failed. Please try again later.\n\n * 500 UpsertDatabaseAdvancedThreatProtectionFailed - An error has occurred while saving Advanced Threat Protection settings, please try again later\n\n * 500 GetDatabaseAdvancedThreatProtectionFailed - Failed to get Advanced Threat Protection settings"
+          },
+          "201": {
+            "description": "Successfully created the database Advanced Threat Protection state.",
+            "schema": {
+              "$ref": "#/definitions/DatabaseAdvancedThreatProtection"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update a database's Advanced Threat Protection state with minimal parameters": {
+            "$ref": "./examples/DatabaseAdvancedThreatProtectionCreateMin.json"
+          },
+          "Update a database's Advanced Threat Protection state with all parameters": {
+            "$ref": "./examples/DatabaseAdvancedThreatProtectionCreateMax.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AdvancedThreatProtectionProperties": {
+      "description": "Properties of an Advanced Threat Protection state.",
+      "required": [
+        "state"
+      ],
+      "type": "object",
+      "properties": {
+        "state": {
```

Comment:
You need a provisioninState property since you want to support long running operations

Reply:
Removed the handling of 201 so this is not a long running operation anymore.